website/docs/discover.md

```diff
@@ -0,0 +1,318 @@
+# Discover Podman Desktop in 10 minutes
+
+Podman Desktop is the central place to work with containers as a developer.
+
+The main purpose of Podman Desktop is to use the Docker, Podman and Kubernetes APIs
+to communicate with containers engines and Kubernetes clusters to provide you a graphical
+and unified interface across all these resources.
+
+It also provides extensions to help you install and configure these resources:
+install the Podman engine locally, install Kubernetes clusters locally,
+configure access to remote Kubernetes clusters, configure access to images registries,
+install various CLIs, and more.
+
+## Architecture
+
+The containers technology is a Linux kernel's technology. On a Linux system, you will be able to run Docker and/or Podman
```

Comment:
Bit of an odd sentence, maybe instead it should be something like..

"Container technology (cgroups + namespaces) are native to Linux and do not require ay virtualization. Windows and macOS provide options for running containers inside of a Linux virtual machine". Or something like that.